windows-installer/Package.wxs

```diff
@@ -35,7 +35,7 @@
 		<!-- Execution of install command-->
 		<SetProperty
 			Id="RunInstallAction"
-			Value='"[%SystemFolder]cmd.exe" /c "cd /d [INSTALLFOLDER] &amp;&amp; bin\ideasy.exe -f install > install.log"'
+			Value='"[%SystemFolder]cmd.exe" /c "mkdir [%USERPROFILE]\.ide &amp;&amp; echo On %date% you accepted the IDEasy License. > [%USERPROFILE]\.ide\.license.agreement &amp;&amp; cd /d [INSTALLFOLDER] &amp;&amp; bin\ideasy.exe -f install > install.log"'
```

Comment:
Looks good, maybe add the license link too in a new line?
`.append(" you accepted the IDEasy license.\n").append(LICENSE_URL);`
https://github.com/devonfw/IDEasy/blob/main/documentation/LICENSE.adoc